Makefile

```
@@ -154,6 +154,14 @@ AWS_VAULT_BIN := $(LOCAL_BIN_PATH)/aws-vault
 $(AWS_VAULT_BIN): $(TOOLS_DIR)/go.mod $(TOOLS_DIR)/go.sum
 	@cd $(TOOLS_DIR) && GOBIN=${LOCAL_BIN_PATH} $(GO) install github.com/99designs/aws-vault/v6
 
+VENV_DIR := $(LOCAL_BIN_PATH)/venv
```

Perhaps something like this to hint at the association with `$(TOOLS_DIR)`?
```suggestion
TOOLS_VENV_DIR := $(LOCAL_BIN_PATH)/tools_venv
```